src/metabase/models/collection/root.clj

```diff
@@ -38,11 +37,12 @@
          :read  perms/collection-read-path
          :write perms/collection-readwrite-path) collection)}))
 
-(extend RootCollection
-  models/IModel
-  (merge
-   models/IModelDefaults
-    {:types {:type :keyword}}))
+;;; TODO FIXME
+#_(extend RootCollection
+    models/IModel
+    (merge
+     models/IModelDefaults
+     {:types {:type :keyword}}))
```

Comment:
what's the story here?

Reply:
Don't need it anymore. Forgot to remove it. Fixed